Basket-Donation-Payments/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const app = express();
+const { resolve } = require("path");
+// This is a sample test API key. Sign in to see examples pre-filled with your key.
+const stripe = require("stripe")("pk_test_51H5xzUCzY7u6lWFYbQg9CjRYyMZ7ONk4AXbcf7fAm2PqLu9afzjv0rSCHQ9zUXnBdSqK8kLs3Xd3bJiy5Easd1xC00d3xbdPOU");
```

Comment:
When you use your real API Key, it should probably be kept secret, I think for server code the common practice is to use linux environment variables.

Reply:
Yep, just fixed that now (hopefully) -- most of this code was sample code